ansible/roles/bootstrap-industrial-edge/README.md

```diff
@@ -1,31 +1,73 @@
-Role Name
-=========
+# Ansible Role: bootstrap-industrial-edge
 
-A brief description of the role goes here.
+This role contains all the imperative tasks that are needed to deploy
+the Industrial Edge validated pattern onto an OpenShift cluster.
 
-Requirements
+## Requirements
 ------------
 
-Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here. For instance, if the role uses the EC2 module, it may be a good idea to mention in this section that the boto package is required.
+* Pre-deployed Openshift or Kubernetes Cluster
+* Must be Cluster Admin to successfully execute this role.
+* There are a few tools that you will need to run this role which are listed below.  
 
-Role Variables
---------------
+| Tool | Description | Download link |
+| ----------- | ----------- | ----------- |
+| kubernetes.core | The collection includes a variety of Ansible content to help automate the management of applications in Kubernetes and OpenShift clusters, as well as the provisioning and maintenance of clusters themselves | **ansible-galaxy collection install kubernetes.core** |
+| Kubernetes Python Cli | The kubernetes.core collection requires the Kubernetes Python client to interact with Kubernetes' APIs. | **pip3 install kubernetes** |
+| Python 3 | Python2 is deprecated from 1st January 2020. Please switch to Python3. | RHEL: <br> **yum -y install rh-python36** |
+| Vault CLI | The Vault CLI is a single static binary. It is a thin wrapper around the HTTP API. Every CLI command maps directly to the HTTP API internally. | RHEL: <br> **sudo yum-config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo**<br> **sudo yum -y install vault** <br><br> Fedora: <br> **sudo dnf config-manager --add-repo https://rpm.releases.hashicorp.com/fedora/hashicorp.repo** <br> **sudo dnf -y install vault** |
```

Comment:
I told the same to William on slack, but I believe this is not correct. We invoke the vault command from inside the vault container, so there is no need for the vault CLI on our laptops.

Reply:
Indeed @mbaldessari. Thanks for clarifying and reviewing this.  I will exclude it from the README file.